src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -90,12 +90,13 @@ function MoneyRequestConfirmPage(props) {
         if (policyExpenseChat) {
             Policy.openDraftWorkspaceRequest(policyExpenseChat.policyID);
         }
-        // Verification to reset billable with a default value, when value in IOU was changed
-        if (typeof props.iou.billable !== 'boolean') {
-            IOU.setMoneyRequestBillable(lodashGet(props.policy, 'defaultBillable', false));
-        }
     }, [isOffline, participants, props.iou.billable, props.policy]);
 
+    const defaultBillable = lodashGet(props.policy, 'defaultBillable', false);
```

Comment:
Use this to avoid putting the whole `props.policy` as a dependency to the `useEffect`. Unless you have other suggestions :)

Reply:
I'm OK with it :D